src/BTCPayServer.Lightning.LND/Base64Url.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+
+namespace BTCPayServer.Lightning.LND
+{
+    public static class Converters
+    {
+        public static string HexStringToBase64UrlString(this string input)
+        {
+            return ToBase64UrlString(input.HexStringToHex());
+        }
+
+        private static string ToBase64UrlString(byte[] arg)
+        {
+            if (arg == null)
+            {
+                throw new ArgumentNullException("arg");
+            }
+
+            var s = Convert.ToBase64String(arg);
```

Comment:
you can use ` Encoders.Base64.EncodeData(arg)`?

Reply:
Thanks, done in [823ba0f](https://github.com/btcpayserver/BTCPayServer.Lightning/pull/64/commits/823ba0f4134a40ca7d2cb6f8b81e565b17f4157e).